lib/assets/stylesheets/modules/_search.scss

```diff
@@ -39,6 +74,7 @@ html.has-search-results-open {
     right: 0;
     bottom: 0;
     z-index: 600;
+    min-height: 100vh;
```

Comment:
On desktop, this line seems to be overridden by line 86. On mobile, turning it off doesn't seem to make any difference. Have I missed what this is supposed to do?